src/evaluations/evaluator.py

```diff
@@ -321,8 +321,8 @@ def save_estimator(self, sketch_estimator_config):
     estimator = sketch_estimator_config.sketch_factory(0)
     sketch_estimator_config_file = os.path.join(
         estimator_dir, ESTIMATOR_CONFIG_FILE)
-    with open(sketch_estimator_config_file, 'wb') as f:
-      pickle.dump(estimator, f)
+    # with open(sketch_estimator_config_file, 'wb') as f:
+    #   pickle.dump(estimator, f)
```

Comment:
why comment this?

Reply:
The pickle module cannot pickle classes that are not defined at the top level of a module. As the sketch estimator config we defined here wraps the voc sketch_factory method which returns a class and is not at the top level, the pickle will fail. The error message reads
```
AttributeError: Can't pickle local object 'VectorOfCounts.get_sketch_factory.**<locals>**.f'
```

I will remove this change in this PR though. Instead, I will create another PR to remove the pickle module since it is not used by the analyzer and report_generator anyway.

FYI: https://docs.python.org/3/library/pickle.html#what-can-be-pickled-and-unpickled